D-lama-service/Test/IntegrationTests/DataPointControllerTest.cs

```diff
@@ -0,0 +1,840 @@
+﻿using d_lama_service.Models.ProjectModels;
```

Comment:
All tests are running and from my point of view, they are also made well. Good job 😃